docs/src/contributing-to-popochiu/basics.md

```diff
@@ -0,0 +1,105 @@
+* **Spread the word**: Host game jams, write guides, create video tutorials, or document your Popochiu-based project on your blog. Sharing your experience with the plugin and creating meaningful content can inspire others and bring new developers - and potential contributors - into the Popochiu community.
+
+## Useful knowledge
+
+1. Popochiu is built with Godot, so please take the time to learn the basics of this game engine and it's GDScript 2 language before opening a PR. Extensive knowledge is great of course, but you can contribute even if you're new to this world. In that case, make sure you read the above paragraphs! It's even more important.
+2. Specifically, knowing how to use Control nodes and wire them with signals is very useful both when coding custom game UIs and to extend the editor plugin. We strongly recommend you learn about this stuff.
+3. Understanding the architecture of the plugin and the engine in broad lines help a lot making sense of its code. Please, review the [Project Overview](project-overview) section for a primer.
+4. If you want to contribute structural or big things, knowledge about design patterns and clean code is a great plus. We can't recommend reading [Game Programming Patterns](http://gameprogrammingpatterns.com/) free online book by Robert Nystrom more! Also, understanding [SOLID principles](https://en.wikipedia.org/wiki/SOLID) makes a huge difference.
+5. Not all is code: the plugin extends the Godot Editor interface in a variety of ways, exposing custom UI. Skills in interaction or UI design, as well as user/developer experience is invaluable to make Popochiu easy to use to everyone.
+6. Last but not least, taking the time to read this whole section of the docs is **very** appreciated.
+
+## Contribution Q&As
+
+**Q:** Is there any guideline for opening a PR?
+**A:** Yes! Please read the [Coding Standards](coding-standards) section for all the relevant information about that.
+
+**Q:** I'm working with an unstable/unsupported version of Godot and I have worked around some errors. Should I open a PR?
+**A:** It depends. If you are using a development or otherwise unstable version of Godot other than a release candidate, then **no**, we will close the PR without even looking at it. The reason is we already spent time addressing bugs that were due to Godot, not Popochiu and that solved themselves once Godot reached a more stable status.  
+If you're testing a release-candidate, we can be interested in reviewing the PR, but you'll have to test it against the next stable release when it's available before it gets merged. Also, the solution should always work on the officialy supported Godot version, if different form the most recent one.
+
+**Q:** Yeah, but since I saw a bug using a newer version, can I at least open an issue so that we don't forget to check it later down the road?
+**A:** **We don't accept bugs related to unsupported version of Godot** anc close them without review. Period.
```

Comment:
Yikes! It sounds a bit harsh, but we can always see how people react before discussing whether it's worth changing our minds.

```suggestion
**A:** **We don't accept bugs related to unsupported versions of Godot**, and close them without review. Period.
```

Reply:
I think I'm going to make this a bit softer, keeping the meaning anyway ;)